mne/evoked.py

```diff
@@ -1965,10 +1965,12 @@ def _get_peak(data, times, tmin=None, tmax=None, mode="abs"):
         The minimum point in time to be considered for peak getting.
     tmax : float | None
         The maximum point in time to be considered for peak getting.
-    mode : {'pos', 'neg', 'abs'}
+    mode : {'pos', 'neg', 'abs', 'min', 'max'}
```

Comment:
confused; this doesn't match the API in the PR description (as was discussed in #12381)

Reply:
Sorry about that! I didn't push my changes yet.